lua/astrocommunity/pack/haskell/init.lua

```diff
@@ -12,33 +12,20 @@ return {
   },
   {
     "mrcjkb/haskell-tools.nvim",
-    ft = { "haskell" },
-    branch = "1.x.x", -- recommended by haskell-tools
-    init = function() astronvim.lsp.skip_setup = utils.list_insert_unique(astronvim.lsp.skip_setup, "hls") end,
-    opts = {
-      hls = {
-        on_attach = function(client, bufnr) require("astronvim.utils.lsp").on_attach(client, bufnr) end,
-      },
-    },
-    config = function(_, opts)
-      local tools = require "haskell-tools"
-      vim.api.nvim_create_autocmd("Filetype", {
-        pattern = "haskell", -- autocmd to start haskell-tools
-        callback = function() tools.start_or_attach(opts) end,
-      })
-
-      vim.api.nvim_create_autocmd("LspAttach", {
-        pattern = "*.hs", -- autocmd to start haskell-tools
-        callback = function(args)
-          local client = vim.lsp.get_client_by_id(args.data.client_id)
-          if client.name == "haskell-tools.nvim" then tools.dap.discover_configurations(args.buf) end
-        end,
-      })
-    end,
     dependencies = {
       "nvim-lua/plenary.nvim",
-      "nvim-telescope/telescope.nvim", -- optional
+      "nvim-telescope/telescope.nvim", -- Optional
```

Comment:
Not directly related to this PR, as it was already there,
but I would suggest not adding the telescope dependency here.
If a user wants telescope uninstalled for some reason, they might be annoyed if it's automatically installed because it's listed as a dependency for haskell-tools.

Reply:
```lua
{
"nvim-telescope/telescope.nvim",
optional = true
}
```
Solves this issue. :)
Reference: https://github.com/folke/lazy.nvim#-plugin-spec

Reply:
Oh, nice. I'll update the snippet in the readme.